doc/community-build/docker/cortx-all/README.md

```diff
@@ -132,9 +133,10 @@ This document provides step-by-step instructions to build required binaries and
     **Example output** 
     ```
     [root@dev-system ~]# docker images --format='{{.Repository}}:{{.Tag}} {{.CreatedAt}}'|grep cortx|grep -v cortx-build
-    cortx-data:2.0.0-0 2022-03-02 07:32:02 -0700 MST
-    cortx-rgw:2.0.0-0 2022-03-02 07:31:31 -0700 MST
-    cortx-all:2.0.0-0 2022-03-02 07:31:23 -0700 MST
+    cortx-data:2.0.0-0 2022-03-23 11:45:02 -0700 MST
+    cortx-rgw:2.0.0-0 2022-03-23 11:45:31 -0700 MST
+    cortx-all:2.0.0-0 2022-03-23 11:45:23 -0700 MST
+    cortx-control:2.0.0-0 2022-03-23 11:51:23 -0700 MST
     ```
 ### Tested by:
 - Feb 08 2022: Amnuay Boottrakoat (amnuay.boottrakoat@seagate.com) on a Windows running VMWare Workstation 16 Player with CentOs 7.9.2009
```

Comment:
@AbhijitPatil1992 , did you test this? If so, you should add yourself to this Tested by section. If not, how are you confident that your changes didn't break anything?